lib/debt_management_center/financial_status_report_service.rb

```diff
@@ -180,7 +184,9 @@ def submit_vha_batch_job(vha_submissions)
       submission_batch = Sidekiq::Batch.new
       submission_batch.on(
         :success,
-        'DebtManagementCenter::FinancialStatusReportService#send_vha_confirmation_email'
+        'Form5655Submission#send_vha_confirmation_email',
+        'email' => @user.email&.downcase,
+        'email_personalization_info' => email_personalization_info
```

Comment:
Is it necessary to call to another class, which then calls back to this module? Seems circular to me

Reply:
Also, the arguments for `Form5655Submission#send_vha_confirmation_email` don't seem to match what you're sending here, so maybe you meant to use the method in this class?

Reply:
Perhaps it's worth testing that the `submit_vha_batch_job` method is calling the proper method with the proper arguments? That's where the hole is, I think

Reply:
@ryan-mcneil Thanks for the review, here's what I'm thinking

> Is it necessary to call to another class, which then calls back to this module? Seems circular to me

Yeah it is, making the base service argument optional with `nil` initialization instead

> Also, the arguments for Form5655Submission#send_vha_confirmation_email don't seem to match what you're sending here, so maybe you meant to use the method in this class?

Both `email` and `email_personalizawtion_info` will be included as part of the options argument within a hash in `Form5655Submission#send_vha_confirmation_email(_status, options)` because of how the callbacks work. More info here: https://github.com/sidekiq/sidekiq/wiki/Batches#callbacks

You could also refer to the form 526 model for their implementation: [batch jobs](https://github.com/department-of-veterans-affairs/vets-api/blob/a2db1b6138a81968e45e253eb418f2bb758d6bf0/app/models/form526_submission.rb#L67-L74) and the [callback](https://github.com/department-of-veterans-affairs/vets-api/blob/a2db1b6138a81968e45e253eb418f2bb758d6bf0/app/models/form526_submission.rb#L312-L322)

Reply:
Re: removing teh use of that other class - nice 👍 . Definitely feels cleaner.

And yes, you're right, `Sidekiq::Batch#on` wraps those extra arguments in the `options` hash. Sorry, I'm back to ruby after years in JS land, and I'm re-familiarizing myself with certain ruby concepts!